_docs/developer-information/contribute-to-drill/020-apache-drill-contribution-ideas.md

```diff
@@ -84,8 +77,8 @@ Initially, concentrate on basics:
 
 Implement custom storage plugins for the following non-Hadoop data sources:
 
-  * NoSQL databases (such as Mongo, Cassandra, Couch etc)
-  * Search engines (such as Solr, Lucidworks, Elastic Search etc)
+  * NoSQL databases (such as Mongo, Couch etc)
+  * Search engines (such as Solr, Lucidworks etc)
   * SQL databases (MySQL< PostGres etc)
   * Generic JDBC/ODBC data sources
   * HTTP URL
```

Comment:
Since this was written, most of these are done... You can query most relational DBs via the JDBC plugin.  We don't have a generic ODBC plugin, so maybe leave that.  We do have an HTTP URL plugin.

Reply:
Sure, I will change back this part, and use another PR for the update of apache-drill-contribution-ideas.md.